protos/flow.proto

```diff
@@ -99,7 +87,6 @@ message CreateTablesFromExistingOutput {
 
 message SyncFlowOptions {
   uint32 batch_size = 1;
-  map<uint32, RelationMessage> relation_message_mapping = 2;
```

Comment:
if we only deprecate this field then we won't break `pause`/`unpause` upgrade

Reply:
will need to retain the other Message types for this, also not sure if it's enough since CDC state also loses fields

Reply:
please deprecate and not delete